packages/cardpay-sdk/sdk/scheduled-payment-module.ts

```diff
@@ -1089,6 +1089,48 @@ export default class ScheduledPaymentModule {
     await waitUntilSchedulePaymentTransactionMined(hubRootUrl, scheduledPaymentId, authToken, options.listener);
   }
 
+  async estimateGasTokenToSchedule(
```

Comment:
Could `estimateSchedulePaymentInGasToken` be a bit more descriptive descriptive name?

Reply:
updated